client/my-sites/google-my-business/stats/index.js

```diff
@@ -220,7 +221,22 @@ class GoogleMyBusinessStats extends Component {
 				<QueryKeyringConnections forceRefresh />
 				<QueryKeyringServices />
 
-				{ ! isLocationVerified && (
+				{ ! locationData && (
+					<Notice
+						status="is-error"
+						text={ translate(
+							'There is an error with your Google My Business Connection. ' +
+								'Please {{a}}contact support{{/a}}',
+							{
+								components: {
+									a: <a href="/help/contact" target="_blank" rel="noopener noreferrer" />,
```

Comment:
We should use the `CALYPSO_CONTACT` constant for the url (an [example](https://github.com/Automattic/wp-calypso/blob/dc338f215ea3730f54eaf5f6856435327be1c85a/client/state/data-layer/wpcom/email-forwarding/get/index.js#L41)). Do we really want to open the support page in a new window though?

Reply:
forgot this existed 😅 I will update, and we do not need a new tab